packages/bitski/src/-private/sdk.ts

```diff
@@ -226,7 +226,7 @@ export class BitskiSDK {
       transactionCallbackUrl: options.callbackURL,
       ...options,
       additionalHeaders: {
-        'X-CLIENT-VERSION': BITSKI_SDK_VERSION,
+        'X-CLIENT-VERSION': 'bitski-sdk-vBITSKI_SDK_VERSION',
```

Comment:
This is probably not what you want

Reply:
This is what was previously there:

```
await replaceInFiles({
  files: ['./lib/**/*', './dist/**/*'],
  from: 'BITSKI_SDK_VERSION',
  to: `"bitski-sdk-v${packageJson.version}"`,
});
```

<img width="1512" alt="Screenshot 2023-12-14 at 7 50 20 PM" src="https://github.com/BitskiCo/bitski-js/assets/4204797/1d43f3db-ff30-46bd-9e39-6a6f346b6733">